packages/calling/calling/package.json

```diff
@@ -61,8 +61,6 @@
     "@types/mocha": "9.0.0",
     "@types/node": "16.11.9",
     "@types/uuid": "8.3.4",
```

Comment:
Why are we removing eslint related plugins from calling. We are also adding calling into .eslintignore. Wouldn't this remove all eslint related checks on calling package?

Reply:
We have these plugins already available in package.json globally present so calling can also use those. And we have added calling .eslintignore so that rules that are there in global .eslintrc.js file are not being run for calling since we have our own .eslintrc.json file which is still being used and performing eslint related checks for calling package